lib/dataTrigger/executeDataTriggers.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * @typedef executeDataTriggers
+ * @param {Document[]} documents
+ * @param {DataTriggerExecutionContext} context
+ * @return {Promise<DataTriggerExecutionResult[]>}
+ */
+function executeDataTriggers(documents, context) {
```

Comment:
should it be placed into verifySTPacket folder?

Reply:
Yeah, makes sense